### DISCUSSION TITLE
TabView component not working in NextJs

### DISCUSSION THREAD

**Discussion by Warisaremou** [primefaces/community#498] [PrimeReact] [answered]
### In the code below I add primeReact TabView component and it's not showing. I'm using NextJs

![code](https://github.com/primefaces/community/assets/107433692/255d3495-b86f-4fa3-838b-2618c881d784)
![Capture d’écran du 2023-11-18 23-47-03](https://github.com/primefaces/community/assets/107433692/2958d696-87cc-4635-a5e5-f646de426114)

**Comment by melloware** [comment]
You didn't mention your PrimeReact version of what your setup of your Theme.css code looks like?

**Reply by Warisaremou** [reply]
## Here is the layout
![code-2](https://github.com/primefaces/community/assets/107433692/f77d84d6-655c-476b-9a69-1c82f4955587)

## And here the version
![WhatsApp Image 2023-11-20 at 08 49 30](https://github.com/primefaces/community/assets/107433692/6ff297cf-3df3-4fed-bca9-2b880d9b07d8)

**Reply by melloware** [reply]
OK I would create a Stackblitz reproducer showing the issue and then open a GitHub ticket with that reproducer.

**Reply by Warisaremou** [reply]
### ok

**Reply by melloware** [reply, marked as answer]
Someone reported it here also as a bug:https://github.com/primefaces/primereact/issues/5395
